dallinger/heroku/clock.py

```diff
@@ -61,19 +42,21 @@ def check_db_for_missing_notifications():
 
             # ask amazon for the status of the assignment
             try:
-                assignment = conn.get_assignment(assignment_id)[0]
+                assignment = mturk.get_assignment(assignment_id)[0]
                 status = assignment.AssignmentStatus
             except:
                 status = None
             print "assignment status from AWS is {}".format(status)
             hit_id = p.hit_id
-
-            # general email settings:
-            # username = os.getenv('dallinger_email_username')
-            # fromaddr = username + "@gmail.com"
-            # email_password = os.getenv("dallinger_email_key")
-            # toaddr = config.get('HIT Configuration', 'contact_email_on_error')
-            whimsical = config.get("whimsical")
+            # Use a null handler for now since Gmail is blocking outgoing email
```

Comment:
Exactly - I wanted to move that problem out of the already complicated code and document it that way. I think another client solved the email delivery problem by using SendGrid, or at least that's how I remember it. I'm not sure if this could be hooked up through the Heroku deployment process. At any rate, yes, keep that ticket open. :)